docs/events.md

```diff
@@ -37,8 +37,8 @@ def send_mail_on_complete_work_item_2(sender, work_item, user, context, **kwargs
 
 ## @on
 
-Right now `@on` is just an alias for `django.dispatch.receiver`. This very likely will change
-in the future, but the public API should stay the same.
+`@on` is just a wrapper for `django.dispatch.receiver` that handles
+deprecations of Calumas custom events as well.
```

Comment:
I'd consider not stating the exact differences between `@on` and `@receiver`.

Reply:
Fixed

Reply:
Done